app/src/popup/components/ExpandedSessionListInput.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Database from "../../engine/Database";
+
+export class ExpandedSessionListInput extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            sessions: [],
+        };
+    }
+
+    componentDidMount() {
+        const storage = new Database();
+        this.setState({sessions: storage.loadSessions()});
+    }
+
+    render() {
+        let options = [];
+        for (const sessionName of this.state.sessions) {
+            options.push(<option value={sessionName}></option>);
+        }
+        return (
+            <>
+                <input list="allSessions" type="text" className="session-name-input" placeholder="..." />

```

Comment:
`placeholder` should be more descriptive, like: *Enter the name of session*.
Probably further releases.